models/models/participant.py

```diff
@@ -126,7 +127,7 @@ class ParticipantUpsert(SMBase):
     """External upsert model for participant"""
 
     id: int | OpenApiGenNoneType = None
-    external_id: str | OpenApiGenNoneType = None
+    external_ids: dict[str, str | OpenApiGenNoneType] | OpenApiGenNoneType = None
```

Comment:
Drawing this to your attention @illusional — is this an appropriate use of `OpenApiGenNoneType`? Want to express that dict entries can be `"foo": None` to remove an entry from the database.

Reply:
Yeah I think this is sweet! We'll find out :)